cypress/integration/homepage/home_page_latest_message_spec.js

```diff
@@ -51,9 +51,11 @@ describe('Tests the Current Message on the Homepage', function () {
     });
   });
 
-  it('Checks card image and, if Bitmovin video, player exists and video autoplays', () => {
+  it('\itmovin video, player exists and video autoplays', () => {
```

Comment:
Not sure this deletion was intentional...

Reply:
done